acceptance/tests/resource/user/should_create.rb

```diff
@@ -16,7 +14,7 @@
   agent.user_get(name)
 
   case agent['platform']
-  when /sles/, /solaris/, /windows/
+  when /sles/, /solaris/, /windows/, /osx/
```

Comment:
While debugging this test I realized that something like: ```user { 'testuser': ensure=>present }``` creates the user ```testuser``` in OSX, but not the group. I'm not sure if this is known or expected behavior, but for now I added OSX to the list of platforms we don't expect to create the group.